sycl/test-e2e/Graph/RecordReplay/add_nodes_after_finalize.cpp

```diff
@@ -1,102 +1,11 @@
 // REQUIRES: level_zero, gpu
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
+// Extra run to check for leaks in Level Zero using ZE_DEBUG
 // RUN: %if ext_oneapi_level_zero %{env ZE_DEBUG=4 %{run} %t.out 2>&1 | FileCheck %s %}
 //
 // CHECK-NOT: LEAK
 
-// This test creates a graph, finalizes it, then continues to add new nodes to
-// the graph with the record & replay API before finalizing and executing the
-// second graph.
-// The second run is to check that there are no leaks reported with the embedded
-// ZE_DEBUG=4 testing capability.
+#define GRAPH_E2E_EXPLICIT
```

Comment:
I think this test should run the Record&Reply version of the test

```suggestion
#define GRAPH_E2E_RECORD_REPLAY
```